source/module_esolver/esolver_ks.h

```diff
@@ -10,6 +10,9 @@
 #include "module_io/cal_test.h"
 #include "module_psi/psi.h"
 
+// // add by jghan for rdmft calculation
+// #include "module_rdmft/rdmft.h"
+
```

Comment:
seems it is not used, so I remind you remove it when your PR is ready for merging

Reply:
I will delete it